data/json/monster_special_attacks/monster_attacks.json

```diff
@@ -614,5 +614,27 @@
     "no_dmg_msg_npc": "%1$s tries to impale <npcname>, but fails to penetrate their armor.",
     "dodgeable": true,
     "blockable": true
+  },
+  {
+    "type": "monster_attack",
+    "attack_type": "melee",
+    "id": "dermatik_impale",
+    "condition": {
+      "and": [
+        { "and": [ { "u_has_flag": "GRAB_FILTER" }, { "npc_has_flag": "GRAB" } ] },
+        {
+          "not": { "or": [ { "u_has_flag": "ACIDBLOOD" }, { "u_has_flag": "PARAIMMUNE" }, { "u_has_effect": "dermatik" } ] }
+        }
+      ]
+    },
+    "effects": [ { "id": "dermatik", "duration": 3600 } ],
```

Comment:
This just adds the effect to the character, not a limb.

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/84619419/184e3b08-424f-42d5-aa41-65972a49db29)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/84619419/f69bd532-e642-461f-b31f-834a60401e3c)